src/panels/lovelace/entity-rows/types.ts

```diff
@@ -14,7 +14,8 @@ export interface ActionRowConfig extends EntityConfig {
   action_name?: string;
 }
 export interface EntityFilterEntityConfig extends EntityConfig {
-  state_filter?: Array<{ key: string } | string>;
+  state_filter?: Array<Condition | LegacyFilterCondition | string | number>;
+  conditions?: Array<Condition | LegacyFilterCondition | string | number>;
```

Comment:
We totally can 😄

In order to make conditional et entity-filter card configs uniform, I made options the same (conditional uses `conditions`), but we totally can let entity-filter with `state_filter` option.

What do you think, stay the same or make them uniform ?

I changed that on 97fffe2891c0cfc60829ffc4719728401ddf0256

Reply:
IMO, it's named "condition" because it renders the card according to "conditions" while here it filters the list of entities according to state. 
It's just a proposal, may be someone else will have another thought about it.